internal/provisioner/plain/manifests/06_deployment.yaml

```diff
@@ -18,7 +18,7 @@ spec:
       serviceAccountName: plain-provisioner-admin
       containers:
         - name: plain-provisioner
-          image: quay.io/operator-framework/plain-provisioner:latest
+          image: quay.io/operator-framework/plain-provisioner:main
```

Comment:
I realize we also default to the latest tag for the plain provisioner's --unpack-image flag: https://github.com/operator-framework/rukpak/blob/main/internal/provisioner/plain/main.go#L66.

Reply:
@timflannagan ah nice thanks for pointing that out. I've added some constants to take care of that https://github.com/operator-framework/rukpak/pull/206/files#diff-00e176bec896af2959cbd2132aedac49f892da315eca7c81de39287295c3605dR56-R60

Reply:
I like the idea of a constant: can we consolidate that into a single variable though?